test/e2e/es_cluster_logging.go

```diff
@@ -196,7 +199,7 @@ func ClusterLevelLoggingWithElasticsearch(c *client.Client) {
 					{
 						Name:    "synth-logger",
```

Comment:
That's the name of the container which I want to be different from the name of the pod.